_includes/footer.html

```diff
@@ -178,7 +157,9 @@ <h2 class="h6">
                         Washington, D.C. 20552
                     </p>
                     <div class="footer_general-inqueries">
-                        <b class="footer_general-inqueries-label">For general inquiries only</b>
+                        <h4 class="footer_general-inqueries-label">
+                            <i>For general inquiries only</i>
```

Comment:
This is still going to use a faux-italic so we should probably remove the `<i>`.